container/Auth/SignUp/SignUpForm.js

```diff
@@ -1,25 +1,32 @@
-import React, { useContext } from 'react';
+import React, { useContext, useRef, useState } from 'react';
 import { useForm, Controller } from 'react-hook-form';
 import { MdLockOpen } from 'react-icons/md';
-import { Input, Switch, Button } from 'antd';
+import { Input, Switch, Button, Select } from 'antd';
 import FormControl from 'components/UI/FormControl/FormControl';
 import { AuthContext } from 'context/AuthProvider';
 import { FieldWrapper, SwitchWrapper, Label } from '../Auth.style';
+import { getDocumentType } from '../../Listing/SignUp/SingUpParams';
+import { FaAngleDown } from 'react-icons/fa';
+import UploadImage from '../UploadImage';
```

Comment:
I think we can remove all related to this on this form because the account will be created with a default image